sdk/durabletask/azure-mgmt-durabletask/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 1.0.0b2 (2025-04-16)
```

Comment:
@berndverst please remember to update date to latest when you are to merge/release.